lib/cadet/factory.ex

```diff
@@ -117,4 +137,19 @@ defmodule Cadet.Factory do
       code: sequence(:code, &"alert(#{&1})")
     }
   end
+
+  def mcq_answer_factory do
+    %{
+      choice_id: Enum.random(0..2)
+    }
+  end
+
+  def library_factory do
+    %{
+      chapter: Enum.random(1..20),
+      globals: Faker.Lorem.words(Enum.random(1..3)),
+      externals: Faker.Lorem.words(Enum.random(1..3)),
+      files: Enum.map(1..5, fn _ -> Faker.File.file_name() end)
```

Comment:
Oooh didn't know you could do that. Done.

Also, there needs to be brackets around `&Faker.File.file_name/0` because of how the macro expansion works:
```
iex > quote(do: &Faker.File.file_name/0 |> Stream.repeatedly() |> Enum.take(5)) |> Macro.to_string

"&(Faker.File.file_name() / 0 |> Stream.repeatedly() |> Enum.take(5))"
```